lib/puppet/util/windows/sid.rb

```diff
@@ -75,7 +75,10 @@ def name_to_principal(name, allow_unresolved = false)
           raw_sid_bytes = sid_ptr.read_array_of_uchar(get_length_sid(sid_ptr))
         end
       rescue => e
-        Puppet.debug("Could not retrieve raw SID bytes from '#{name}': #{e.message}")
+        # Avoid debug logs pollution with valid account names
+        # https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--1300-1699-
```

Comment:
I would remove this link since we rely on the `ERROR_INVALID_SID_STRUCTURE` constant which is already defined in puppet.